src/constants/button.js

```diff
@@ -1,12 +1,15 @@
 /* @flow */
 export const BUTTON_LABEL = {
-    CONNECT: ('connect' : 'connect')
+    CONNECT:     ('connect' : 'connect')
 };
 
 export const BUTTON_COLOR = {
     GOLD:        ('gold' : 'gold'),
```

Comment:
Should we remove GOLD from this list? I remember hearing that it's not supported since we don't want this button to get confused with the PayPal payments button.